packages/vite/src/node/plugins/manifest.ts

```diff
@@ -32,13 +32,52 @@ export function manifestPlugin(config: ResolvedConfig): Plugin {
     return perEnvironmentPlugin('native:manifest', (environment) => {
       if (!environment.config.build.manifest) return false
 
-      return nativeManifestPlugin({
-        root: environment.config.root,
-        outPath:
-          environment.config.build.manifest === true
-            ? '.vite/manifest.json'
-            : environment.config.build.manifest,
-      })
+      const root = environment.config.root
+      const outPath =
+        environment.config.build.manifest === true
+          ? '.vite/manifest.json'
+          : environment.config.build.manifest
+
+      function getChunkName(chunk: OutputChunk) {
+        if (chunk.facadeModuleId) {
+          return normalizePath(
+            path.relative(root, chunk.facadeModuleId),
+          ).replace(/\0/g, '')
+        }
+        return `_${path.basename(chunk.fileName)}`
```

Comment:
```suggestion
        return (
          getChunkOriginalFileName(chunk, root, false) ??
          `_${path.basename(chunk.fileName)}`
        )
```
I think this is better because it would be more easier to understand whether the implementation of this function is different from the one in the non-native one.